xmlWorksheet.go

```diff
@@ -20,6 +20,7 @@ import (
 // http://schemas.openxmlformats.org/spreadsheetml/2006/main.
 type xlsxWorksheet struct {
 	mu                     sync.Mutex
+	formulaSharedIndex     sync.Map
```

Comment:
When changing cell formula or remove column and row, the cached cell should be removed.

Reply:
I updated this pull request, clear cache on remove column and row.